docs/click_feature.js

```diff
@@ -91,8 +91,10 @@ function createHtmlForPopup(feature, rowData, scoreParameters, columns) {
     if (value && column.endsWith(' PERCENTAGE')) {
       value = parseFloat(value).toFixed(3);
     }
-    property.innerText = column + ': ' +
-        (value !== null && value !== undefined ? value : '(data unavailable)');
+    property.innerHTML = column + ': ' +
+        (value !== null && value !== undefined ?
+             value :
+             '<span class="data-unavailable-span">(data unavailable)</span>');
```

Comment:
could just do the styling in the html since we're already having some html in here. Up to you!

Reply:
Prefer to put it in separate file in case we switch here to creating the span via `document.createElement`, or we want to re-style the span.